README.md

```diff
@@ -43,21 +43,21 @@ npm install
 
 ## New Token Methods
 
-Líf token is ERC20 compatible but it also has two more methods to allow the transference of data between users/contracts.
+Líf token is ERC20 compatible but it also has two more methods to allow the transference of data and execution of calls between users/contracts.
 
 ### transferData
 
 Transfer tokens from one address to another.
 ```
-transferData(address _to, uint _value, string _data) => (bool success)
+transferData(address to, uint value, bytes data, bool doCall) => (bool success)
```

Comment:
if doCall is true it will execute the bytes that are sent on a call, if not it will just trigger the event, so this function can be used to register something on the blockchain, mostly signatures, or execute calls between token holders, we are near the gas limit on contract deployment so adding another function to do almost the same I think is not possible.
And yes, we will need to explain a bit more about what doCall can do.